doc/logging_and_error_handling.md

```diff
@@ -4,12 +4,12 @@ We chose to use Timbre for logging because it is the only library that comes wit
 logging in one piece. Aside from that it offers to implement appenders easily and seems heavy used throughout
 the clojure community. You can use file logging and structured logging from built-in appenders.
 
-As a library datahike should not force any logging implementation on the user so Timbre does offer a wide
-variety of options. It can do it itself with a simple edn configuration or you can use your preferred Java
+As a library, Datahike should not force any logging implementation on the user so Timbre does offer a wide
+variety of options. It can do it itself with a simple edn configuration, or you can use your preferred Java
```

Comment:
maybe `Timbre can do the config via a simple edn configuration, or you can use your preferred Java logging implementation.`?